frontend/src/metabase/visualizations/components/Visualization.jsx

```diff
@@ -553,3 +580,44 @@ export default class Visualization extends React.PureComponent {
     );
   }
 }
+
+function isSamePosition(question, clicked, previousClicked) {
```

Comment:
What I want to basically achieve here is to compare whether `clicked` and `previousClicked` is the same, but since `clicked` isn't easily serialized, it can't be easily compared via `_.isEqual`.